scenario/LiquidationBotScenario.ts

```diff
@@ -505,7 +604,7 @@ scenario(
     upgrade: {
       targetReserves: exp(20_000, 18)
     },
-    filter: async (ctx) => ctx.world.base.network === 'mainnet',
+    filter: async (ctx) => matchesDeployment(ctx, [{deployment: 'mainnet'}]), // XXX enable for Polygon
```

Comment:
This scenario is challenging on Polygon. You need to take out a flash loan large enough that attempting to swap that amount will result in huge price slippage. Taking out a loan of that size if challenging on Polygon, however, since the pools available to provide flash loan are relatively small.

It may be possible to thread that needle at current prices/liquidity, but it seems very likely this test would start failing in the near future when prices or liquidity change; not sure what would be best here.

Reply:
Shouldn't the small market size mean you just have to take out a smaller loan for the huge slippage to occur?

Reply:
sure, but it's still a fairly tight window between an amount that is big enough to cause sufficient slippage and the max amount of the flash loan you can borrow.

right now, swapping 35 WBTC (~$700K USDC) will cause sufficient slippage; but the pool we're using for flash loans only has $1M of USDC to borrow.

we can set a value right now that is within that window, but it wouldn't take a huge price increase in WBTC or a huge decrease in the pool liquidity before the scenario starts failing